.argo/test-yamls/pod-throughput-test.yaml

```diff
@@ -4,7 +4,7 @@ name: quick-exit
 description: does nothing and exits quickly
 resources:
   mem_mib: 32
-  cpu_cores: 0.1
+  cpu_cores: 0.01
```

Comment:
Is this for test only? Such low cpu request could potentially trigger kernel issues.

Reply:
This is for test only. We still have num pods <= 80 per node and we do not set cpu limit